### PR TITLE
feat: appearance of selected elements

### LINKED ISSUE #57: [REQUEST] Active Tab Color
**Describe the feature you'd like to see implemented**

I am sure this has been actively discussed, but I found it hard to notice which tab I was actually looking at

imageimage

So what about making the active tab use the light blue color we have.
That would also add some color to the window.

the left color 
image

Maybe something like this
image

A bit like discord does, just using our own colors
image

**Additional context**

Discord username (if different from GitHub): DerL30N#3751

### PR DESCRIPTION
**Pull request by MisterChocker**
Part of #57

## Summary of Changes

- Added SVG for current version selection
- ~~Add SVG for current tab (aircraft/settings) selection~~ (scrubbed for now)

## Additional context

Discord username (if different from GitHub):

### CODE REVIEW

File: src/renderer/components/AircraftSection/TrackSelector.tsx [hunk 1]
```diff
@@ -54,7 +54,21 @@ export const Track = styled(BaseTrack)`
...
+      border-left-color: ${props => props.isSelected ? `${colors.cardSelected}` : `${colors.cardBackgroundHover}`};
+  }
+  
+  transition: border-color linear 200ms;
+  transition: border-top-color linear 200ms;
+  transition: border-right-color linear 200ms;
+  transition: border-bottom-color linear 200ms;
+  transition: border-left-color linear 200ms;
+
 `;
```
Comment (marcsoler) [inline src/renderer/components/AircraftSection/TrackSelector.tsx:72]: How about this CSS trick:

```
  border: ${props => props.isSelected ? `solid transparent 2px` : 'solid transparent 2px'};
  :after {
    content:'';
    position: absolute;
    top:5%;
    left:-2px;
    width: 5px;
    height: 90%;
    background-color: ${props => props.isSelected ? `${colors.cardSelected}` : `transparent` } ;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
  }
```

Result:
image

Comment (MisterChocker) [reply src/renderer/components/AircraftSection/TrackSelector.tsx:72]: Thanks for the suggestion, I am going to check this out. I knew there is a css solution but I have zero experience, so highly appreciated!

**Comment by Snapmatics** [comment]
Can you make it so it covers all the side?

**Comment by MisterChocker** [comment]
> Can you make it so it covers all the side?

All the side?

**Comment by FoxtrotSierra6829** [comment]
> 
> 
> > Can you make it so it covers all the side?
> 
> All the side?

Guess he means it should be 100% height, which would also require increasing width to keep it symmetric.

**Comment by MisterChocker** [comment]
Well I personally like it the way it currently is. Do we need a poll for that?

**Comment by marcsoler** [comment]
Let me know if you would like a hover/active animation on that (similar to discord)

**Comment by MisterChocker** [comment]
> Let me know if you would like a hover/active animation on that (similar to discord)

That would be an awesome addition! I really like that idea

**Pull request closed without merging**